src/Command/CreateEntity/User/BulkUserCreationCommand.php

```diff
@@ -0,0 +1,233 @@
+                )
+            );
+        }
+        if ($inputUserPrefix) {
+            $this->initializeUserPrefixOption($inputUserPrefix);
+        }
+        if ($inputBatchSize) {
+            $this->initializeBatchOption($inputBatchSize);
+        }
+    }
+
+    /**
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->symfonyStyle->comment('Executing Bulk user creation');
+
+        try {
+            $processDataResult = $this->bulkCreateUsersService->createUsers(
+                $this->lineItemIds,
+                $this->lineItemSlugs,
+                $this->userPrefix,
+                $this->batchSize,
+                $input->getOption(self::OPTION_GROUP_PREFIX)
+            );
+            if (!empty($processDataResult->getNonExistingLineItems())) {
+                $this->symfonyStyle->note(
+                    sprintf(
+                        'Line Items with slugs/ids \'%s\' were not found in the system.',
+                        implode(',', $processDataResult->getNonExistingLineItems()),
+                    )
+                );
+            }
+            $this->symfonyStyle->success(
+                sprintf('%s', $processDataResult->getMessage())
+            );
+        } catch (Throwable $exception) {
+            $this->symfonyStyle->error($exception->getMessage());
+
+            return 1;
+        }
+
+        return 0;
+    }
+
+    private function initializeLineItemIdsOption(string $inputLineItemIds): void
+    {
+        $lineItemIds = array_filter(
+            explode(',', $inputLineItemIds),
+            static function (string $value): bool {
+                return !empty($value) && (int)$value > 0;
+            }
+        );
+        if (empty($lineItemIds)) {
+            throw new InvalidArgumentException(
+                sprintf('Invalid %s option value received.', self::OPTION_LINE_ITEM_IDS)
+            );
+        }
+
+        $this->lineItemIds = array_map('intval', $lineItemIds);
+    }
+
+    private function initializeLineItemSlugsOption(string $inputLineItemSlugs): void
+    {
+        if (!empty($inputLineItemSlugs)) {
+            $this->lineItemSlugs = array_filter(
+                explode(',', $inputLineItemSlugs),
+                static function (string $value): bool {
+                    return !empty($value);
+                }
+            );
+        }
+        if (empty($this->lineItemSlugs)) {
+            throw new InvalidArgumentException(
+                sprintf('Invalid %s option received.', self::OPTION_LINE_ITEM_SLUGS)
+            );
+        }
+    }
+
+    private function initializeUserPrefixOption(string $inputUserPrefix): void
+    {
+        $this->userPrefix = array_filter(
+            explode(',', (string)$inputUserPrefix),
+            static function (string $value): bool {
+                return !empty($value);
+            }
```

Comment:
not needed

Reply:
Above mentioned method is used to remove empty comma separated values in a user prefix options

Reply:
yes, I see that. without static function empty values will be considered as false